WordPress/Classes/ViewRelated/Post/Scheduling/LightNavigationController.swift

```diff
@@ -0,0 +1,44 @@
+import Foundation
+
+// A Navigation Controller with a light navigation bar style
+class LightNavigationController: UINavigationController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupBarAppearance()
+    }
+
+    private func setupBarAppearance() {
+
+        let separatorColor: UIColor
+
+        if #available(iOS 13.0, *) {
+            separatorColor = .systemGray4
+        } else {
+            separatorColor = .lightGray
+        }
+
+        let navigationBarAppearanceProxy = UINavigationBar.appearance(whenContainedInInstancesOf: [LightNavigationController.self])
```

Comment:
@frosty Sorry about that, missed this on the first pass. Just to be clear, this works because `UIAppearance` is added when the view is added to the _window_. So `viewDidAppear` is too late, but `viewWillAppear` and earlier is theoretically ok.

Still, I absolutely agree with you. It should be added in AppDelegate. I've added to `WPStyleGuide+ApplicationStyles` but could also add to an extension file since `LightNavigationController` isn't necessarily needed in the share or draft extensions. Let me know if you want me to move it out of those targets.